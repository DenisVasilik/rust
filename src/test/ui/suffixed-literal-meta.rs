#![feature(custom_attribute)]

#[macro_use(x = 1usize)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1u8)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1u16)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1u32)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1u64)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1isize)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1i8)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1i16)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1i32)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1i64)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1.0f32)] //~ ERROR: suffixed literals are not allowed in attributes
#[macro_use(x = 1.0f64)] //~ ERROR: suffixed literals are not allowed in attributes
fn main() { }
