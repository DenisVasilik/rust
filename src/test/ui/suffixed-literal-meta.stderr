error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:3:17
   |
LL | #[macro_use(x = 1usize)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:4:17
   |
LL | #[macro_use(x = 1u8)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:5:17
   |
LL | #[macro_use(x = 1u16)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:6:17
   |
LL | #[macro_use(x = 1u32)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:7:17
   |
LL | #[macro_use(x = 1u64)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:8:17
   |
LL | #[macro_use(x = 1isize)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:9:17
   |
LL | #[macro_use(x = 1i8)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:10:17
   |
LL | #[macro_use(x = 1i16)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:11:17
   |
LL | #[macro_use(x = 1i32)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:12:17
   |
LL | #[macro_use(x = 1i64)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:13:17
   |
LL | #[macro_use(x = 1.0f32)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: suffixed literals are not allowed in attributes
  --> $DIR/suffixed-literal-meta.rs:14:17
   |
LL | #[macro_use(x = 1.0f64)] //~ ERROR: suffixed literals are not allowed in attributes
   |                 ^^^^^^
   |
   = help: instead of using a suffixed literal (1u8, 1.0f32, etc.), use an unsuffixed version (1, 1.0, etc.).

error: aborting due to 12 previous errors

